cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -676,6 +688,20 @@ protected void configureRoutes() {
 
         addProjectLocksHandlers("/project-locks/{name}", requiredRoles);
         addProjectLockRightsHandlers("/project-lock-rights/{project-id}", requiredRoles);
+
+
+        addUserManagementHandlers();
```

Comment:
I think you're source branch needs updating. these are already in the target branch

Reply:
Yes.  thank you for catching this, I will revisit.  I think the user management was merged after my branch started.  I did rebase but maybe this was missed.